docs/modules/data-structures/pages/managing-map-memory.adoc

```diff
@@ -281,30 +282,33 @@ As well as the method parameters, you can also use the `map.setTTL()` method to
+Forced eviction is an automatic process that Hazelcast uses to remove data from the HD Memory Store. When your eviction policy does not free enough native memory, operations that add entries to a map such as `map.put()` trigger the forced eviction process. 
 
-If you are using Hazelcast Enterprise, Hazelcast can use forced eviction to remove more entries before throwing an exception.
+You cannot configure forced eviction. It is either enabled or disabled, depending on how your map or cache is configured.   
 
-NOTE: Hazelcast can only use this feature for maps whose in-memory format is set to `NATIVE`.
+For a map, forced eviction is enabled only if the map is configured with the following:
 
-The forced eviction mechanism tries to free more memory in the following order:
+- An <<policy, eviction policy>> other than `NONE`
+- The xref:setting-data-format.adoc[`NATIVE` in-memory format]
 
-* When the normal eviction is not enough, forced eviction is triggered and first it tries to evict approx. 20% of the entries from the current partition. It retries this five times.
-* If the result of above step is still not enough, forced eviction applies the above step to all maps. This time it might perform eviction from some other partitions too, provided that they are owned by the same thread.
-* If that is still not enough to free up your memory, it evicts not the 20% but all the entries from the current partition.
-* If that is not enough, it will evict all the entries from the other data structures; from the partitions owned by the local thread.
+For a cache, forced eviction is enabled when you use the xref:setting-data-format.adoc[`NATIVE` in-memory format].
 
-Finally, when all the above steps are not enough, Hazelcast throws a native `OutOfMemoryException`.
+The forced eviction process tries to free memory in the following order:
 
-When you have an evictable cache/map, you should safely write entries to it without facing with any memory shortages. Forced eviction helps to achieve this. Regular eviction removes one entry at a time while forced eviction can remove multiple entries, which can even be owned by other caches/maps.
+. Evict 20% of entries in the current partition for the map or cache that triggered the forced eviction process. Hazelcast retries the operation that triggered the forced eviction process up to five times. If the operation throws a native out-of-memory exception on the fifth try, see step 2.
+. Evict 20% of entries for all maps or caches that meet the conditions for forced eviction and that are in a partition that is owned by the same partition thread that triggered the forced eviction process.
+. Remove the remaining 80% of entries in the current partition for the map or cache that triggered the forced eviction process.
+. Remove all entries from maps or caches that meet the conditions for forced eviction and that are in a partition that is owned by the same partition thread that triggered the forced eviction process.
```

Comment:
Found this part to verbose, details in implementation can be changed at any time and this part can become quickly stale.

Reply:
I agree. If you're happy, we can remove this process.